docs/develop/node/intro/types-of-node.md

```diff
@@ -0,0 +1,16 @@
+---
+id: types-of-node
+title: Types of Node
+sidebar_label: Types of Node
+---
+
+You can run three different types of node - Validator Node, RPC Node, and Archival Node.
+
+## Validator Node
+Validator Nodes are the operators of the NEAR blockchain and are essential to the health of the network. Validator nodes receive new states and blocks while participating in block validation, and verify blocks and states.
```

Comment:
Note that nonvalidator nodes also verify blocks. I think it is better to say something like "Validator nodes participate in the consensus and produce blocks and/or chunks".